### PR TITLE
[nasa/nos3#288] Added additional folders to APPLICATION_PLATFORM_INC_LIST

### PR DESCRIPTION
**Pull request by jlucas9**
Resolves issue with the latest RTS format.

### CODE REVIEW

**Review by kevincbruce — APPROVED** [review]
Looks good to me! Once the merge is pushed to Gitlab I'll merge and check if it resolves all issues.

**Review by dccutrig — APPROVED** [review]
Looks good to me!

**Review by rjbrown2 — APPROVED** [review]

**Pull request merged**
